lib/composers/with_redux.js

```diff
@@ -0,0 +1,30 @@
+import compose from '../compose';
+
+export default function composeWithRedux(fn, L, E, options) {
+  const onPropsChange = (props, onData) => {
+    const context = typeof props.context === 'function' ? props.context() : props.context;
```

Comment:
Here's the problem in finding the store. Getting it from the options works fine for testing. But, we can have that.
But, the getting it from the context is something common for mantra, so this won't suite here.
So,here's my suggestion.

Let's pass a function as the second argument. If that's an object, we can consider it as the store. If it's a function, at runtime we'll pass props to it and that should return the store. 
What do you think?